lms/validation/_lti_launch_params.py

```diff
@@ -51,7 +51,7 @@ def _decode_jwt(self, data, **_kwargs):
         return params
 
 
-class _CommonLTILaunchSchema(LTIV11CoreSchema):
+class CommonLTILaunchSchema(LTIV11CoreSchema):
```

Comment:
I tried merging both but as LTIV11CoreSchema is used on the `validation/auth` side of things and only having one changes the behavior there.

Reply:
I meant that the class could be in this file, rather than merged. Currently there's `_lti_launch_params.py` which contains nearly everything and then just once class outside. If all the sub-classes were here, this would still be private.